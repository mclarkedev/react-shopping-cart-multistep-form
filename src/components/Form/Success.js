import React from 'react'

const Success = (props) => {
    return (
        <div>
            <h2>Order Submitted!</h2>
        </div>
    )
}

export default Success
