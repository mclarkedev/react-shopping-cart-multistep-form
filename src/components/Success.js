import React from 'react';
import SuccessContainer from '../styles/elements/success';

import Fade from 'react-reveal/Fade';

const Success = () => {
    return (
        <Fade>
            <SuccessContainer>
                <h1>Order Submitted!</h1> 
                <p>You will receive an email receipt within the next 15 minutes.</p>    
            </SuccessContainer>
        </Fade>
    )
}

export default Success;