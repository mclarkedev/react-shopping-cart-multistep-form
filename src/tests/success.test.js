import React from 'react';
import { shallow, mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import Success from '../components/Success';

describe('Success Page', () => {
    it('should be reachable by url', () => {
        const wrapper = mount(
            <MemoryRouter initialEntries={[ '/success ']}>
                <Success />
            </MemoryRouter>
        );
        expect(wrapper.find(Success)).toHaveLength(1);
    })

    it('should render without errors', () => {
        const wrapper = shallow(<Success />);
        expect(wrapper).toMatchSnapshot();
    });
});