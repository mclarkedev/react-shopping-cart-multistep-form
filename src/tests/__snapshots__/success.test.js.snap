// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success Page should render without errors 1`] = `ShallowWrapper {}`;
